server/chart_config.json

```diff
@@ -937,6 +937,46 @@
       "unit": "%"
     }
   },
+  {
+    "category": "Housing",
+    "title": "Housing Unit Value",
```

Comment:
Housing Units by Value

Reply:
Done. Add change other places to "Units"